src/samples.rs

```diff
@@ -0,0 +1,91 @@
+use std::collections::BTreeMap;
+
+use reddsa::frost::{
+    redjubjub::{
+        keys::{
+            KeyPackage, PublicKeyPackage, SecretShare, SigningShare,
+            VerifiableSecretSharingCommitment, VerifyingShare,
+        },
+        round1::{NonceCommitment, SigningCommitments},
+        round2::SignatureShare,
+        Identifier, SigningPackage, VerifyingKey,
+    },
+    redpallas::{
+        frost::{Element, Scalar},
+        Ciphersuite, Field, Group,
+    },
+};
+
+use reddsa::frost::redjubjub::JubjubBlake2b512;
+
+type C = JubjubBlake2b512;
+
+fn element1() -> Element<C> {
+    <C as Ciphersuite>::Group::generator()
```

Comment:
Note that this introduces quite a lot of bias in tests. Can't use use random point instead?